### ISSUE TITLE
crashed loading cudnn

### ISSUE THREAD

**Issue by DefinitlyEvil** [ibab/tensorflow-wavenet#214]
```WARNING:tensorflow:From D:\Temp\tensorflow-wavenet\train.py:266 in main.: initialize_all_variables (from tensorflow.python.ops.variables) is deprecated and will be removed after 2017-03-02.
Instructions for updating:
Use `tf.global_variables_initializer` instead.
Trying to restore saved checkpoints from .\logdir\train\2017-02-06T00-00-41 ... No checkpoint found.
files length: 3
E c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\cuda\cuda_dnn.cc:385] could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR
E c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\cuda\cuda_dnn.cc:352] could not destroy cudnn handle: CUDNN_STATUS_BAD_PARAM
F c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\kernels\conv_ops.cc:532] Check failed: stream->parent()->GetConvolveAlgorithms(&algorithms)
```

**Comment by akademi4eg** [comment]
Seems like Tensorflow/CuDNN/Windows problem. Are you sure there is no conflict of versions? Are you able to run other tensorflow-based applications?

**Comment by DefinitlyEvil** [comment]
Yea, I can use TFLearn.

**Comment by DefinitlyEvil** [comment]
any ideas?

**Comment by DefinitlyEvil** [comment]
I got CUDA Toolkit v8.0 and CUdnn v5.1, I found that I installed both `tensorflow` and `tensorflow-gpu` package so I uninstalled both and re-installed only `tensforflow-gpu` with the latest 0.12.1 version. still crashed, seemed c++ crash not python script error.

**Comment by akademi4eg** [comment]
Unfortunately, I don't have windows setup. So can't help much.
In this thread https://github.com/tensorflow/tensorflow/issues/2033 someone had same error and it was suggested that it is caused by some error in code.
"files length: 3" suggests that you are using some custom corpus. Can you try if it would work on VCTK or could you attach those sound files so that I can try training on them?

**Comment by DefinitlyEvil** [comment]
@amademi4eg I used first 3 pieces from Bach's French suite (I am majoring music composing but I like ML)

**Comment by Lef-F** [comment]
@DefinityEvil I had the same issue. I uninstalled tf 0.12 and installed 1.0.0rc2 and with some minor tweaks in the code it works now!

**Comment by DefinitlyEvil** [comment]
@Lef-F thx, gonna try

**Comment by lemonzi** [comment]
See https://github.com/tensorflow/tensorflow/issues/6606. There seems to be a bug in 0.12 indeed. @Lef-F, which tweaks did you need to perform besides the logits and labels in the cross-entropy?
